src/appengine/private/components/testcase-detail/testcase-detail.html

```diff
@@ -430,6 +430,12 @@
                                   version="[[info.testcase.impact_beta_version]]"
                                   likely="[[info.testcase.impact_beta_version_likely]]"></impact-button>
                             </template>
+                            <template is="dom-if" if="[[info.testcase.impact_head_version]]">
+                              <impact-button
+                                  release="Head"
+                                  version="[[info.testcase.impact_head_version]]"
+                                  likely="[[info.testcase.impact_head_version_likely]]"></impact-button>
```

Comment:
Is this actual useful information that someone using the UI would want to see / query by? I wonder if we can just omit this to keep things simple.

Reply:
OK, I've removed. I wasn't sure if you had a policy that all known information should be displayed in the ClusterFuzz UI for completion's sake.